
sim:
  dt: 0.2
  dt_in_sim: 0.2
  image:
    output_image: true
    height: 1500
    width: 400
    m2pix: 100 # pixels per meter
    y_baseline: 100 # pixels up at line y = 0
  world:
    road:
      width: 2.0
  our_duckie:
    start_pose: [0.0, 0.0, 0] # [x,y, theta]
    velocity: 1.0
    radius: 0.25
    type: "our_duckie"
  other_duckie:
    start_pose: [0.0, 5.0, 0] # [x,y, theta]
    velocity: 0.25
    radius: 0.25
    type: "slow_duckie" # Available: slow_duckie.     To be implemented: unstable_duckie, ...
